aten/src/ATen/native/mps/operations/Activation.mm

```diff
@@ -1486,7 +1486,7 @@ Tensor glu_backward_mps (const Tensor& grad_output,
               newCachedGraph = new CachedGraph(mpsGraph);
               MPSGraphTensor* inputTensor = mpsGraphRankedPlaceHolder(mpsGraph, self);
 
-              MPSGraphTensor* betaTensor = mpsGraphScalarPlaceHolder(mpsGraph, beta);
+              MPSGraphTensor* betaTensor = mpsGraphScalarPlaceHolder(mpsGraph, getMPSScalarType(ScalarType::Float));
```

Comment:
A new implementation of `mpsGraphScalarPlaceHolder()` is not needed. Instead, can you please use the existing function and just pass ScalarType::Float to it?
`MPSGraphTensor* betaTensor = mpsGraphScalarPlaceHolder(mpsGraph, ScalarType::Float);`